hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/cluster/ClusteringPlanPartitionFilterMode.java

```diff
@@ -18,12 +18,31 @@
 
 package org.apache.hudi.table.action.cluster;
 
+import org.apache.hudi.common.config.EnumDefault;
+import org.apache.hudi.common.config.EnumDescription;
+import org.apache.hudi.common.config.EnumFieldDescription;
+
 /**
  * Clustering partition filter mode
  */
+@EnumDescription("Partition filter mode used in the creation of clustering plan.")
 public enum ClusteringPlanPartitionFilterMode {
```

Comment:
Really changed these, the documentation was pretty rough and had to read the code to figure out what is happening